addons/pinniped/config-controller/controllers/v1_controller.go

```diff
@@ -93,7 +92,16 @@ func (c *PinnipedV1Controller) Reconcile(ctx context.Context, req ctrl.Request)
 	cluster := clusterapiv1beta1.Cluster{}
 	if err := c.client.Get(ctx, req.NamespacedName, &cluster); err != nil {
 		if k8serror.IsNotFound(err) {
-			// TODO: Reconcile delete if cluster not found
+			if k8serror.IsNotFound(err) {
```

Comment:
`k8serror.IsNotFound(err) {` duplicated?

Reply:
yup, thanks for pointing that out! fixed it.